tnc/data_handler.py

```diff
@@ -56,7 +53,7 @@ def __init__(self) -> None:
         self.transmission_uuid = ""
 
         # Received my callsign crc if we received a crc for another ssid
-        self.received_mycall_crc = b""
+        self.received_mycall_crc = b""  # Does this need to be a class variable?
```

Comment:
I'm not sure about its usage right now. My intention was, using class wide variables for callsigns instead of using static.XXXX while an ongoing arq transmission. But might it be possible, this variable is unused?

Reply:
It's set in three places:
`__init__` on line 56,
`arq_data_received` on line 448,
`arq_cleanup` on line 2288

It's never used anywhere, only set.

Reply:
Then we can remove it, I think 🙂

Reply:
Removed...